gamemode/modules/base/cl_jobmodels.lua

```diff
@@ -14,7 +37,7 @@ function DarkRP.setPreferredJobModel(teamNr, model)
     local job = RPExtraTeams[teamNr]
     if not job then return end
     preferredModels[job.command] = model
-    sql.Query(string.format([[REPLACE INTO darkp_playermodels VALUES(%s, %s);]], sql.SQLStr(job.command), sql.SQLStr(model)))
```

Comment:
There's another issue. After the table is migrated to have three columns, the old version of this query will fail with this error: `table darkp_playermodels has 3 columns but 2 values were supplied`.

Because SQLite errors are silent (the query just returns `false`, and you have to print `sql.LastError()` to get the error,  job models are silently no longer saved.

This would not have a problem if the query was `REPLACE INTO darkp_playermodels(jobcmd, model) VALUES(%s, %s);`, but this is not something we can fix in hindsight :thinking: